hicrep/hicrep.py

```diff
@@ -46,13 +46,13 @@ def sccOfDiag(diag1: np.ndarray, diag2: np.ndarray):
     iDiagNZ2 = diag2[idxNZ]
     # variance stabilizing xformation -- these are for geometric
     # average of the variances only
-    iDiagVS1 = vstran(iDiagNZ1)
-    iDiagVS2 = vstran(iDiagNZ2)
+    iDiagVS1 = (len(iDiagNZ1) + 1)/(12 * len(iDiagNZ1))
+    iDiagVS2 = (len(iDiagNZ2) + 1)/(12 * len(iDiagNZ2))
     rho = np.corrcoef(iDiagNZ1, iDiagNZ2)[0, 1]
     # the original R implementation impose Bessel's correction
     # in the variance calculation, which I don't think it make sense
     # here
-    ws = iN * np.sqrt(np.var(iDiagVS1, ddof=1)*np.var(iDiagVS2, ddof=1))
+    ws = iN * np.sqrt(iDiagVS1 * iDiagVS2)
```

Comment:
The variance of `1/n, 2/n, ..., n/n` should be `(1 - 1.0/n**2)/12.0` -- can you verify? Also, can you make the variance calculation as a separate function and call it something like `varVstran()` and annotate its relation with the R's vstran() implementation? We should also rename `iDiagVS1` to `iDiagVarRank1` or somethinkg like that since it's not longer

Reply:
Good catch - I was incorrectly calculating sample variance.

Reply:
Actually I think the sample variance was what was being calculated before, as var was called with ddof=1 . Changing it alters the scc values very slightly from what the R version has.